live-site/lib/data.ts

```diff
@@ -9,46 +8,45 @@ const liveSiteTabInfo: TabInfo[] = [
   { name: 'Team', link: '#team' }
 ];
 
-
-
+const TeamInfo: TeamProps[] = [
+  { name: 'Dhivas', points: 75 },
+  { name: 'Shoosh', points: 23 },
+  { name: 'Shlosh', points: 1 }
```

Comment:
Let's add 2 more teams since we have 5 teams and change the names to be the actual team names (to make sure that they fit the lines and things like that)
<img width="742" alt="Screenshot 2023-01-18 at 1 23 00 PM" src="https://user-images.githubusercontent.com/59738880/213263202-e1f248f4-84d6-472a-8ddd-b7f513727801.png">